### DISCUSSION TITLE
Which repo hosts the Organic Maps' location share website (omaps. app/{{ge0_code}}/{{name}})?

### DISCUSSION THREAD

**Discussion by reinhart1010** [organicmaps/organicmaps#1670] [Q&A] [answered]
Hello, I'm thinking of slightly improving the web preview of the Organic Maps' "ge0.me" location link previews. For example, https://omaps.app/ctfiTcGWj5/Republic_of_Pandora

However, I still can't find the right repo to have a look and contribute to the code behind it.

**Comment by biodranik** [comment, marked as answer]
Hi and thanks for the interest! We forgot to make it public )
Here is it: https://github.com/organicmaps/url-processor
